conmon/ctr_logging.c

```diff
@@ -74,10 +76,12 @@ void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, char *cuu
 		nexit("Include journald in compilation path to log to systemd journal");
 #endif
 		cuuid = cuuid_;
+		if (strlen(cuuid) <= TRUNCIDLEN)
+			nexit("Container ID is too short");
+		strncpy(short_cuuid, cuuid_, TRUNCIDLEN);
```

Comment:
`strncpy` won't add a '\0' if it is not in the first TRUNCIDLEN bytes of cuuid_.  Please add it explicitly with `short_cuuid[TRUNCIDLEN] = '\0';`

Reply:
fixed!